tools/docker-network/config.docker.json

```diff
@@ -79,5 +79,10 @@
   },
   "manaInitializer": {
     "FaucetAPI": "http://faucet:8080"
+  },
+  "blockIssuer": {
+    "rateSetter": {
+      "mode": "disabled"
```

Comment:
This overrides the other PR https://github.com/iotaledger/goshimmer/pull/2540/files#diff-3543443f8fa8c99ddae5adfe4ef6ba6b7231d434ce983f6fe5e72bf3e12a8ebe in which the rateSetter.enabled param is removed, thus activating the default "deficit" rateSetter.
What should we do here? Running the "defaults" or disabling it?

Reply:
I think locally we should disable it now, as it causes some problems with EvilSpammer. Later we can re-enable when other issues are resolved and we can focus on fine-tuning the rate-setter.